test/functional/editor_controller_test.rb

```diff
@@ -45,8 +45,10 @@ def setup
         tags: 'question:question,one',
         template: 'question',
         redirect: 'question'
-    assert_select "h3", "Ask a question of the community"
-    assert_select "input#taginput[value=?]", "question:question,one"
+        assert_response :redirect
```

Comment:
Oh, just to be clear, if it matches level of indent of the line below it that'd be perfect.